### PR TITLE
Use mountOnEnter and unmountOnExit for PF Tabs

### PR DESCRIPTION
**Pull request by israel-hdez**
Removing workaround to render DOM only if tabs are visible.
Workaround was due a missing feature in older versions of PF4.

Closes kiali/kiali#1716.

### CODE REVIEW

File: src/pages/IstioConfigDetails/IstioConfigDetailsPage.tsx [hunk 1]
```diff
@@ -376,14 +376,14 @@ class IstioConfigDetailsPage extends React.Component<RouteComponentProps<IstioCo
     if (this.hasOverview()) {
       tabs.push(
         <Tab key="istio-overview" title="Overview" eventKey={0}>
-          {this.state.currentTab === 'overview' ? this.renderOverview() : undefined}
+          {this.renderOverview()}
```
Comment (mtho11) [inline src/pages/IstioConfigDetails/IstioConfigDetailsPage.tsx]: This reads much nicer.

File: src/pages/ServiceDetails/ServiceDetailsPage.tsx [hunk 2]
```diff
@@ -356,46 +356,39 @@ class ServiceDetails extends React.Component<ServiceDetailsProps, ServiceDetails
...
       <Tab eventKey={1} title="Traffic" key="Traffic">
-        {currentTab === 'traffic' && (
-          <TrafficDetails
-            trafficData={this.state.trafficData}
-            itemType={MetricsObjectTypes.SERVICE}
-            namespace={this.props.match.params.namespace}
-            serviceName={this.props.match.params.service}
-          />
-        )}
+        <TrafficDetails
```
Comment (mtho11) [inline src/pages/ServiceDetails/ServiceDetailsPage.tsx]: +1

File: src/components/Tab/Tabs.tsx [hunk 3]
```diff
@@ -97,6 +99,8 @@ export default class ParameterizedTabs extends React.Component<TabsProps> {
             this.tabTransitionHandler(ek as number);
           }
         }}
+        mountOnEnter={this.props.mountOnEnter}
+        unmountOnExit={this.props.unmountOnExit}
```
[the remark this answers is not in the document]
Comment (israel-hdez) [reply src/components/Tab/Tabs.tsx]: Sounds reasonable, given that all usages use "true" here.
OK, I'll change this, but I'll leave the explicit values in usages :)

[the remark this answers is not in the document]
Comment (israel-hdez) [reply src/components/Tab/Tabs.tsx]: Done!

**Pull request merged**